.github/workflows/cd-dev.yaml

```
@@ -0,0 +1,33 @@
+name: "Continuous Deployment to dev"
+on:
+  push:
+    branches:
+      - dev
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Short SHA
+        uses: benjlevesque/short-sha@v1.2
+        id: short-sha
+        with:
+          length: 8
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.GITHUB_ACTOR }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push to GitHub Packages
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: |
+            ghcr.io/Ingsis-Group-4/snippet-permission:${{ steps.short-sha.outputs.sha }}
```

```suggestion
            ghcr.io/Ingsis-Group-4/snippet-permission-dev:${{ steps.short-sha.outputs.sha }}
```
We should have different tags for dev and main